### PR TITLE
Automatically convert tabular data to csv

### PR DESCRIPTION
**Pull request by enricorotundo**

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -97,6 +104,15 @@ graph:
     path: /inputs
   outputs:
   - path: /outputs
+- id: convert-table-to-csv-node
+  job_id: convert-table-to-csv-job
+  inputs:
+  - node_id: metadata-node
+    predicate: '.*'
```
Comment (philwinder) [inline config.yaml:111]: If you're not predicating, then you don't need to be connected to the metadata node. Remove this input to run in parallel.

**Review by philwinder — DISMISSED** [review]

File: containers/frictionless-extract/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM python:3.9.16
+RUN pip install frictionless==5.12.1 frictionless[excel] frictionless[parquet]
```
[the remark this answers is not in the document]
Comment (enricorotundo) [reply containers/frictionless-extract/Dockerfile:2]: done

File: containers/frictionless-extract/test.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+checkFileExists() {
+    if [ ! -f "$1" ]; then
+        echo "File $1 does not exist"
+        exit 1
+    fi
+}
+
+main() {
+    rm -rf $SCRIPT_DIR/outputs
+    docker run -it --rm -v $SCRIPT_DIR/../test/testdata/various_tables:/inputs -v $SCRIPT_DIR/outputs:/outputs  --entrypoint "" $IMAGE run
```
[the remark this answers is not in the document]
Comment (enricorotundo) [reply containers/frictionless-extract/test.sh:25]: it already uses `file` to do that https://github.com/bacalhau-project/amplify/blob/2806943d205e976f74e3922e79402c6a86630d4f/containers/scripts/run_program.sh#L33

[the remark this answers is not in the document]
Comment (enricorotundo) [reply containers/frictionless-extract/test.sh:33]: added a specific test in 2f99e63ab9670873a85e24f76b2755542c6fb994 just in case

**Pull request merged**